README.md

```diff
@@ -85,7 +85,7 @@ A simple disko configuration may look like this:
 If you'd saved this configuration in /tmp/disko-config.nix, and wanted to create a disk named /dev/nvme0n1, you would run the following command to partition, format and mount the disk.
 
 ```
-$ sudo nix run github:nix-community/disko -- --mode zap_create_mount /tmp/disko-config.nix --arg disks '[ "/dev/nvme0n1" ]'
+$ sudo nix run github:nix-community/disko -- --mode disko /tmp/disko-config.nix --arg disks '[ "/dev/nvme0n1" ]'
```

Comment:
Do we have some documentation what each of these modes do?
It's not exactly intuitive from the name.

Reply:
added some documentation to the help output of the disko cli